nucypher/blockchain/eth/deployers.py

```diff
@@ -789,28 +788,29 @@ def make_agent(self) -> EthereumContractAgent:
     def allocation_registry(self):
         return self.__allocation_registry
 
-    def assign_beneficiary(self, beneficiary_address: str) -> dict: 
+    @validate_checksum_address
+    def assign_beneficiary(self, checksum_address: str) -> dict:
         """Relinquish ownership of a UserEscrow deployment to the beneficiary"""
-        if not is_checksum_address(beneficiary_address):
-            raise self.ContractDeploymentError("{} is not a valid checksum address.".format(beneficiary_address))
+        checksum_address = checksum_address
```

Comment:
Is this doing something?

Reply:
It's apparently assigning itself to itself.

Reply:
It's the effect of a rename plus a rebase 😅 
But it's fixed in a subsequent PR so I preferred to leave that as is.

Reply:
...

Reply:
✅